doc/fvwm3_manpage_source.adoc

```diff
@@ -5051,7 +5051,7 @@ styles starting with "Application".
 	other than the default or to set the window manager default styles.
 +
 _stylename_ can be a window's name, class, visible name, or resource
-string. It may contain the wildcards '*' and '?', which are matched in
+string. It may contain the wildcards '\*' and '?', which are matched in
```

Comment:
Another option is to use `{asterisk}` instead of escaping it, as sometimes escaping has issues with asciidoctor.